tools/DocsLinter/Program.cs

```diff
@@ -195,8 +195,15 @@ private static void Main(string[] args)
 
             try
             {
-                var request = WebRequest.CreateHttp(remoteLink.Url);
+                var strippedUrl = remoteLink.Url;
```

Comment:
What is this drive by change fixing?

Reply:
well, if you would have waited until it's out of WIP, I would have added the reason :P
what happened with .NET Core 2.1: it will fail to retrieve URLs that have an anchor. Previously it simply ignored the anchor